frontend/src/components/common/header/header.tsx

```diff
@@ -0,0 +1,46 @@
+import { useSelector } from 'hooks/hooks';
+import { AppRoute, ButtonType } from 'common/enums/enums';
+import { RootState } from '../../../common/types/types';
+import { Button, Link } from '../common';
+import defaultAvatar  from '../../../assets/img/default-user-avatar.svg';
+import styles from './header.module.css';
+
+const Header: React.FC = () => {
+  const { user } = useSelector(({ auth }: RootState) => ({
+    user: auth.user,
+  }));
+
+  const hasUser = Boolean(user);
+
+  return (
+    <header className={styles.container}>
+      <div className={styles.logoContainer}>
+        <div className={styles.logoContainerLogo}></div>
+        <span>Logo</span>
+      </div>
+
+      <ul className={styles.navigation}>
+        <li className={` ${styles.liNavigation} ${styles.activeLiNavigation}`}>Podcasts</li>
```

Comment:
```suggestion
        <li className={styles.liNavigation}>Podcasts</li>
```
Don't need it for now
Also don't forget to remove styles